packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/reducer.js

```diff
@@ -24,10 +29,40 @@ const reducer = (state, action) =>
   // eslint-disable-next-line consistent-return
   produce(state, draftState => {
     switch (action.type) {
+      case 'ADD_URLS_TO_FILES_TO_DOWNLOAD': {
```

Comment:
It seems that you add urls to files to upload not to download. Am I wrong?

`draftState.filesToUpload = ....` <- Add something to files to upload.